### DISCUSSION TITLE
Error at msnoise cc stack -m

### DISCUSSION THREAD

**Discussion by Olango1** [ROBelgium/MSNoise#370] [Q&A] [answered]
Hello,

I have been trying to stack so as to finally  compute_mwcs but I keep running into the error below.

  File "/home/miniforge3/lib/python3.10/site-packages/msnoise/scripts/msnoise.py", line 908, in cc_stack
    main('mov', loglevel=loglevel)
  File "/home/miniforge3/lib/python3.10/site-packages/msnoise/s04_stack2.py", line 178, in main
    mov_rolling, mov_sample = mov_stack
ValueError: too many values to unpack (expected 2)

Kindly assist.

Best,
Emerald

**Comment by ThomasLecocq** [comment, marked as answer]
looks like you are using an old version of the database with a newer version of the msnoise code?  did you install "master" from github?  In that case, you need to check the mov_stack parameter, its structure has changed, former ``1,2`` is now ``(('1d','1d'),('2d','1d'))``
